### PR TITLE
use correct ObjectMapper in Index[IO/Merger] in AggregationTestHelper

### PR DESCRIPTION
**Pull request by himanshug**
or else exceptions are printed in the test because ObjectMapper present in  IndexIO can't deserialize the Sketch aggregators in the segment metadata.
and set isInputThetaSketch to false for merged aggregator, it doesn't change anything but false is more appropriate.

### CODE REVIEW

**Comment by drcrallen** [comment]
I don't understand the merged aggregator change, can you explain a little more about why false is more appropriate?

File: extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeAggregatorFactory.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
           name,
           Math.max(size, castedOther.size),
           shouldFinalize,
-          true,
+          false,
```
Comment (nishantmonu51) [inline extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeAggregatorFactory.java]: why this change ?

Comment (himanshug) [inline extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeAggregatorFactory.java]: isInputThetaSketch is used only at indexing time when reading user's input data to decide if its [base64 encoded] sketch objects. it has no role to play afterwards.
that value in merged factory does not change anything as long as it stays same, just thought false would be better to store there. it could stay true as well without any problems.

**Comment by himanshug** [comment]
@drcrallen @nishantmonu51 ?

**Comment by gianm** [comment]
👍  lgtm

**Comment by drcrallen** [comment]
👍

**Comment by nishantmonu51** [comment]
:+1:
